src/test/resources/transformation/output/application-transformed.json

```diff
@@ -1,5 +1,5 @@
 {
-  "applicant1ScreenHasMarriageBroken": "Yes",
+  "applicant1HasMarriageBroken": ["marriageBroken"],
   "applicant2ScreenHasMarriageBroken": "No",
```

Comment:
so applicant1HasMarriageBroken and applicant1HasCivilPartnershipBroken has now become Set. What about applicant2ScreenHasMarriageBroken which is used for joint applications ?

Reply:
Spoke to Lalita about this and the outcome was another ticket eventually to do this functionality.